script/release

```diff
@@ -99,6 +96,6 @@ x = STDIN.gets
 exit(1) unless x.strip == 'yes'
 puts
 
-verbose("git push origin")
-verbose("git push origin -f #{tag}")
-verbose("git push origin -f #{semvertag}")
+verbose("git push origin HEAD:release --follow-tags -f")
+
+puts "Remember to tag the release with #{semvertag}"
```

Comment:
Should this script create the PR with the tag?

Reply:
I can remove it. Just a reminder to use which tag to tag the release when drafting new release.